pages/index.html

```diff
@@ -1,12 +1,41 @@
+<!--must include your name somewhere-->
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Lizet Tovar's Portfolio</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <div class="main-container">
+    <div>
```

Comment:
It is a convention to section a page as `header`, `body`, then `footer`. I think this sectioning would remove the need for this extra `div`